hawkular-command-gateway-war/src/main/java/org/hawkular/cmdgw/ws/server/FeedListenerGenerator.java

```diff
@@ -101,34 +97,9 @@ public void addListeners(String feedId) throws Exception {
         MsgLogger.LOG.infoAddingListenersForFeed(feedId);
 
         MessageProcessor messageProcessor = new MessageProcessor();
-        String messageSelector = String.format("%s = '%s'", Constants.HEADER_FEEDID, feedId);
```

Comment:
In 4db0e03 the queue listeners reside in `BusQueueProcessors`. JMS is quite new to me and I might have misunderstood some concepts. Could you please have a look at `BusQueueProcessors`?